index.js

```diff
@@ -17,6 +17,26 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+    eturn Recipe.insertMany(data) // Iteration 3
+  })
+  .then((result) => {
+    result.forEach(element => console.log(element.title)) // Iteration 3
+    return Recipe.findOneAndUpdate({
+      title: "Rigatoni alla Genovese"
+    },
+      {
+        duration: 100
+      })
+  })
```

Comment:
Here you could have added the `options object` (the 3rd argument for that method) at the end to see the result after updating the documento on the DB